src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -706,6 +706,11 @@ public with sharing class CRLP_RollupUI_SVC {
             // CMT data will generate a unique RecordName to use for the new record.
             CRLP_RollupCMT.Rollup rollupCMTObj = (CRLP_RollupCMT.Rollup) JSON.deserialize(rollupCMT, CRLP_RollupCMT.Rollup.class);
 
+            Boolean isRollupStateEnabled = new CRLP_ApiService().isRollupStateEnabled();
```

Comment:
The abstraction of the logic is helpful, though I would (referring to comment below) say that the `shouldMarkRollupStale()` method might either not exist in this context or be renamed `shouldTriggerRollupChanged()`. I think it would be best if the save methods simply call the `rollupChangedTrigger` (or `filterRuleChangedTrigger`) and leave the logic that checks to see if a _trigger` (i.e., the bridge) is implemented in those methods..

Reply:
I agree "stale" concept is not known within NPSP. In order to avoid ambiguity and since the Bridge is not using triggers to process stale records, perhaps we can keep the name generic and implement details in the method calling the integration API? I have updated the suggested code for the method name: `shouldSendChangeEvent()`. 
I agree there might be better to check `isRollupStateEnabled` in the lower levels closer to calling DSO API, if MetadataAPI can be changed. However, we do not want to differentiate the rollup with all existing rollups if the `isRollupStateEnabled` is false, so there might a need to keep the flag in this class. We still need to traverse existing rollups to determine if the target field is duplicate, but nothing more if the `isRollupStateEnabled` is false.